include/revng/Model/Type.h

```diff
@@ -33,6 +35,11 @@ class model::Type : public model::generated::Type {
 public:
   bool verify(bool Assert = false) const debug_function;
   RecursiveCoroutine<bool> verify(VerifyHelper &VH) const;
+
+public:
+  bool operator==(const model::Type &Another) const {
+    return (*this <=> Another) == std::strong_ordering::equal;
+  }
```

Comment:
Why do we need this?

Reply:
Because otherwise the compiler complains when instantiating a set/map.

`Operator<=>` went in slightly undercooked.